cordova-lib/src/plugman/fetch.js

```diff
@@ -132,8 +132,9 @@ function fetchPlugin(plugin_src, plugins_dir, options) {
                 ));
             }
             // If not found in local search path, fetch from the registry.
-            var splitVersion = plugin_src.split('@');
-            var newID = pluginMapperotn[splitVersion[0]];
+            var splitVersion = cordovaUtil.parseRegistryPluginSpec(plugin_src);
+            var oldId = splitVersion[0] || plugin_src;
+            var newID = pluginMapperotn[oldId];
```

Comment:
Since `oldId` could be either package id or scope + package id i'm not sure how this will work with plugin mapper. Should we use `extractPluginId` instead of `parseRegistryPluginSpec` in L135?